src/app/components/Tab/TabList.tsx

```diff
@@ -0,0 +1,14 @@
+import { Tab as HeadlessUiTab } from "@headlessui/react";
+import { classNames } from "~/app/utils/index";
+
+type Props = React.HTMLAttributes<HTMLDivElement> & {
```

Comment:
This Props accepts all values from React.HTMLAttributes<HTMLDivElement> but it is only passing className to HeadlesssUiTab.List. Shouldn't you pass all the props? otherwise I would change the props just to have a className field and not inherit from the above interface.

Reply:
I thought in case we need to use any attributes; we could just add it as props.